addons/wowl/static/src/types.ts

```diff
@@ -65,6 +65,7 @@ export interface SessionInfo {
 
 export interface Odoo {
   session_info: SessionInfo;
+  debug: string;
```

Comment:
any reason why it is a string and not a boolean? I don't think we want to make a difference between assets and just debug

Reply:
In legacy, it sometimes differentiates the assets mode from the default debug mode.
Maybe we could use  'isDebug(type?:string)'  which returns true if we are in debug mode of the type in param ?